### PR TITLE
e2e test for changes on post purchase page

### LINKED ISSUE #74326: e2e test for purchasing 1 plugin
As part of our testing efforts (p1678135347895659-slack-CKZHG0QCR), we'd like to explicitly break this into an issue to account for the time needed.

Acceptance criteria:

- [ ] Add e2e test for purchasing 1 plugin

### PR DESCRIPTION
**Pull request by lupus2k**
Fixes https://github.com/Automattic/wp-calypso/issues/74326

## Proposed Changes

The main idea is to add e2e test for purchasing 1 plugin, but as we currently don't have the correct Atomic test user ([mentioned here](https://github.com/Automattic/wp-calypso/pull/74746#issuecomment-1479967942)), I added some validations on our changes on the post-purchase page.

- [x] Validate the new copy
- [x] Check for the expiration text
- [x] Validate the ~~three~~ two cards on the bottom section (removed 24/7 card because of [this PR](https://github.com/Automattic/wp-calypso/pull/74796))

## Testing Instructions

* On `/packages/calypso-e2e`, run 'yarn build`
* On Calypso root, run `yarn workspace wp-e2e-tests test -- plugins/plugins__install`

## Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [x] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [x] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [ ] ~~Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)~~
- [ ] ~~Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?~~
- [ ] ~~For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?~~

### CODE REVIEW

File: packages/calypso-e2e/src/lib/pages/plugins-page.ts [hunk 1]
```diff
@@ -307,8 +308,22 @@ export class PluginsPage {
...
+		await this.page
+			.getByRole( 'heading', { name: "Congrats on your site's new superpowers!" } )
+			.click();
+
+		// Check for expiration text
+		await this.page.locator( selectors.installedPluginCard ).getByText( 'expire' ).waitFor();
+
+		// Check for the correct footer cards
+		await this.page.locator( selectors.installedfooterCards ).getByText( 'Keep growing' ).waitFor();
+		await this.page.locator( selectors.installedfooterCards ).getByText( 'Learn More' ).waitFor();
```
Comment (DustyReagan) [inline packages/calypso-e2e/src/lib/pages/plugins-page.ts:320]: Looks good! Should we preemptively remove this check due to [this PR](https://github.com/Automattic/wp-calypso/pull/74796)?

**Review by DustyReagan — APPROVED** [review]

File: packages/calypso-e2e/src/lib/pages/plugins-page.ts (hunk 1 above)
Comment (lupus2k) [reply packages/calypso-e2e/src/lib/pages/plugins-page.ts:320]: Yes, I'll do it, thanks!

**Comment by lupus2k** [comment]
For purchases, we should follow the same steps as [this PR](https://github.com/Automattic/wp-calypso/pull/74746) but using an Atomic user with a credit card without creating a site

**Review by candy02058912 — APPROVED** [review]
LGTM, tests are passing!

**Pull request merged**